program/idl.json

```diff
@@ -7515,6 +7515,225 @@
           }
         ]
       },
+      {
+        "kind": "instructionNode",
+        "name": "initializePausableConfig",
+        "docs": [
+          "Initialize a new mint with the `Pausable` extension.",
+          "",
+          "Fails if the mint has already been initialized, so must be called before `InitializeMint`."
+        ],
+        "optionalAccountStrategy": "programId",
+        "accounts": [
+          {
+            "kind": "instructionAccountNode",
+            "name": "mint",
+            "isWritable": true,
+            "isSigner": false,
+            "isOptional": false,
+            "docs": ["The mint."]
+          }
+        ],
+        "arguments": [
+          {
+            "kind": "instructionArgumentNode",
+            "name": "discriminator",
+            "defaultValueStrategy": "omitted",
+            "docs": [],
+            "type": {
+              "kind": "numberTypeNode",
+              "format": "u8",
+              "endian": "le"
+            },
+            "defaultValue": {
+              "kind": "numberValueNode",
+              "number": 44
+            }
+          },
+          {
+            "kind": "instructionArgumentNode",
+            "name": "pausableDiscriminator",
+            "defaultValueStrategy": "omitted",
+            "docs": [],
+            "type": {
+              "kind": "numberTypeNode",
+              "format": "u8",
+              "endian": "le"
+            },
+            "defaultValue": {
+              "kind": "numberValueNode",
+              "number": 0
+            }
+          },
+          {
+            "kind": "instructionArgumentNode",
+            "name": "authority",
+            "docs": ["The authority that can pause and resume the mint."],
+            "type": {
+              "kind": "zeroableOptionTypeNode",
+              "item": {
+                "kind": "publicKeyTypeNode"
+              }
+            }
```

Comment:
Technically, this is just a public key, but it shouldn't make a difference -- the program will fail if provided a `None`